dbbackup/management/commands/dbbackup.py

```diff
@@ -43,16 +45,29 @@ def handle(self, **options):
         self.compress = options.get('compress')
         self.encrypt = options.get('encrypt')
         self.storage = BaseStorage.storage_factory()
-        database_keys = (self.database,) if self.database else dbbackup_settings.DATABASES
-        for database_key in database_keys:
-            database = settings.DATABASES[database_key]
-            self.dbcommands = DBCommands(database)
+        if options.get('mongo'):
+            # Mongo specific handling.
+            # TODO: Add support for multiple mongo db defined in settings.
```

Comment:
It's not because you're using a mongodb in your django app, that you're using django_mongodb_engine.

I can add in a different PR support for MongoDB as an engine, yes but that wasn't really where this PR was going in the first place.

But I've already said that in previous post on this conversation as it was the first thing I was asked...